### DISCUSSION TITLE
guidance on mediaproxy?

### DISCUSSION THREAD

**Discussion by animegrafmays** [misskey-dev/misskey#9123] [Q&A]
hey friends. I'm trying to set up a new instance and use Mediaproxy, which for the most part functions. but it will not proxy MP4 files from any instance. the error it throws is:

`ERR 1    [server]        StatusError: Rejected type`

I checked google and there's no documentation on specifying types to proxy, and it's only MP4 files that it rejects. Any ideas?

thank you!

**Comment by CustardFlan** [comment]
Comment the next lines in /misskey-media-proxy/built/index.js

//        else if (!file.mime.startsWith('image/') || !FILE_TYPE_BROWSERSAFE.includes(file.mime)) {
//           throw new StatusError('Rejected type', 403, 'Rejected type');
//        }

Works for my deployment
